### PR TITLE
Real image challenge 2016 - Ilayaraja M

### PR DESCRIPTION
**Pull request by ilayaraja89**
Hello Real image,

Please do review the solution and let me know your feedback.

-- 
Ilayaraja M
ilayaraja.edu@gmail.com

### CODE REVIEW

**Comment by ilayaraja89** [comment]
Hi @sudhirj @sivachandran 

Did you had any chance of reviewing this pull request ?

File: docs/Approach.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		...
+	},
+	...
+	...
+}
+```
+2. When a distributor or sub-distributor is created, the permissions are cloned from the base permission object.
+3. Based on the INCLUDE & EXCLUDE inputs, update the permission object of the distributor. Here is the sample 
+4. If the distributor has a parent (if it is a sub-distributor), the INCLUDE & EXCLUDE inputs will be validated against the permission of immediate parent distributor.
+5. A distributor is said to have access to a COUNTRY, if all the cities of all the provinces have permission value to true
```
Comment (sivachandran) [inline docs/Approach.md]: This is not what the problem statement specifies. A distributor can have country but may not have access to specific province or state. Can you change your solution to handle this scenario?

File: docs/Approach.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+6. A distributor is said to have access to a PROVINCE, if all the cities of that province have permission value to true
```
Comment (sivachandran) [inline docs/Approach.md]: The above comment applies here too.

File: docs/Approach.md (hunk 1 above)
Comment (ilayaraja89) [inline docs/Approach.md]: Hi @sivachandran, It is fixed in this commit 79900be. Can you please review ?

**Comment by ilayaraja89** [comment]
Hi @sivachandran @sudhirj 

Sorry for a log gap to address the reviews. I have added one commit  79900be to fix the mentioned issues. Please review and let me know.

**Pull request closed without merging**